src/style/app.css

```diff
@@ -1,3 +1,218 @@
-h1 {
-  color: aquamarine;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&family=Roboto+Slab:wght@300;400;600;700;900&family=Ubuntu:ital,wght@0,400;0,700;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Roboto+Slab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.flex {
+  display: flex;
+}
+
+/* navbar */
+.header {
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: solid 1px #e8e8e8;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+}
+
+.nav-container {
+  align-items: center;
+  justify-content: center;
+  gap: 2.5rem;
+}
+
+.nav {
+  gap: 2.5rem;
+}
+
+.logo {
+  width: 15rem;
```

Comment:
- [ ] Kindly as the requirement says that the Styling should be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly make sure your design looks the same, check the difference bellow

Your design | The template
---|--
<img width="1440" alt="image" src="https://github.com/enmanuelbayen/bookstore-website/assets/79058364/967ce28c-c490-4d46-a0b4-33ab94005a26"> | <img width="1440" alt="image" src="https://github.com/enmanuelbayen/bookstore-website/assets/79058364/b21a97cc-d7df-40b1-a16b-6fe048f5a61a">

- [ ] Use the same margin, buttons styles, colors, space between element, in breaf make sure your design look exactly like template